linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -331,7 +333,7 @@ where
         }
         .with(block),
     );
-    make_certificate(committee, worker, value)
+    make_certificate(committee, worker, value.try_into().expect("Confirmed cert"))
```

Comment:
Here we should also be able to avoid the `expect`, using the appropriate constructor.

Even if inside that constructor we still `expect` for now (but I hope we can remove that entirely soon) it would at least clean up all the call sites.
(But maybe that's something for another PR. :shrug:)

Reply:
Yes, let's leave it for a separate PR - I'm currently in the process of removing a lot of `HashedCertificateValue` usage and maybe it'll go away entirely soon.